package/kedro_viz/models/experiment_tracking.py

```diff
@@ -67,14 +66,15 @@ class TrackingDatasetModel:
 
     dataset_name: str
     # dataset is the actual dataset instance, whereas dataset_type is a string.
-    # e.g. "kedro.extras.datasets.tracking.metrics_dataset.MetricsDataSet"
+    # e.g. "kedro_datasets.tracking.metrics_dataset.MetricsDataSet"
     dataset: AbstractVersionedDataSet
     dataset_type: str = field(init=False)
     # runs is a mapping from run_id to loaded data.
     runs: Dict[str, Any] = field(init=False, default_factory=dict)
 
     def __post_init__(self):
         self.dataset_type = get_dataset_type(self.dataset)
```

Comment:
Is this variable actually still used anywhere now in experiment tracking?

Reply:
Yes, I tried removing this which caused errors. I think it's used for the schema in graphql.

Reply:
I suspect what's happening here is we're passing the full `self.dataset_type` to the frontend, which is then just being turned back into the abbreviated form - see https://github.com/kedro-org/kedro-viz/pull/1214/files#r1071164866.

The full dataset type is used in the frontend in metadata panel in the flowchart view but not in experiment tracking (I think). So there's a good simplification we could make here if all we really need in experiment tracking is the full dataset type.

Feel free to leave as it is for now, but should be an easy tidy up for someone to do in a follow up PR if I'm right.